### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by errolnj** [marioparaschiv/nitro-sniper#364] [Q&A]
Do I need a billing payment/source to claim nitro gifts now? While I was looking through the console logs on replit I saw one that said "Unknown Response: Billing Payment is invalid can't redeem gift" I forgot what it exactly said, but it is something like that.

**Comment by MrD4rio** [comment]
That usually happens with the promotional codes, normal nitro gifts do not necessarly require a payment method to be claimed.
